app/Models/Message.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace Gladiator\Models;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Message extends Model
+{
+    /**
+     * Array of available message types.
+     *
+     * @var array
+     */
+    protected static $types = [
```

Comment:
yeah, I'm fine with the array on the model, but I think we should just start with `reminder` for now. it seems unlikely that will be different from other `reminder` emails